Cython/Compiler/Nodes.py

```diff
@@ -3917,6 +3926,12 @@ def generate_keyword_unpacking_code(self, min_positional_args, max_positional_ar
                     else:
                         code.putln('case %2d:' % i)
                 pystring_cname = code.intern_identifier(arg.name)
+                if arg.pos_only:
+                    if i == last_required_posonly_arg:
+                        code.put_goto(argtuple_error_label)
+                    if i == last_required_arg:
```

Comment:
What if `last_required_arg == `last_required_posonly_arg`? Wouldn't that generate useless code here?